client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -308,18 +326,20 @@ class ListItem extends PureComponent {
               <span className="list-item__author">{`Added by: ${authorName}`}</span>
             </span>
           </label>
-          {!isOrdered && (
+          {!isOrdered && isMember && (
             <VotingBox
               isVoted={isVoted}
               voteForItem={voteForItem}
               votesCount={votesCount}
             />
           )}
-          <button
-            className="list-item__icon z-index-high"
-            onClick={this.handleItemToggling(authorId, _id, isOrdered)}
-            type="button"
-          />
+          {isMember && (
```

Comment:
consider to leave this button visible, but disallow toggling. Otherwise This green "done" indicator won't be visible for viewers. This button isn't necessary but it helps to distinguish list's items.